tests/acceptance/TestHelpers/UploadHelper.php

```diff
@@ -91,56 +89,16 @@ public static function upload(
 				null,
 				$isGivenStep
 			);
-		} else {
-			//prepare chunking
-			$chunks = self::chunkFile($source, $noOfChunks);
-			$chunkingId = 'chunking-' . \rand(1000, 9999);
-			$v2ChunksDestination = '/uploads/' . $user . '/' . $chunkingId;
 		}
 
+		//prepare chunking
```

Comment:
what was done in this preparation?
```suggestion
		//prepare chunking
```

Reply:
`$chunk` &rarr;  Returns array of file chunks(source file split into multiple chunks)
`$chunkingId` &rarr; Generates random identifier (e.g., "chunking-4567") and groups related chunks together
`$result` &rarr; Initializes variable to store upload response which will be updated during chunk upload process